pkg/policies/branch/branch.go

```diff
@@ -455,6 +472,10 @@ func fix(ctx context.Context, rep repositories, c *github.Client,
 			EnforceAdmins:        p.EnforceAdmins.Enabled,
```

Comment:
Need to set this above as well
```
				// No existing protection, create from config.
				afp := !mc.BlockForce
				pr := &github.ProtectionRequest{
					AllowForcePushes: &afp,
				}
```

Reply:
Ah good catch! Fixed in 2cbc17cb002f77b3260b407bc8ca273fc14abb0d, and also added a regression test 15c4e0712cbe6094c1196d30cff836e25aefc825